packages/expo-crypto/ios/CryptoModule.swift

```diff
@@ -47,6 +49,20 @@ private func digestString(algorithm: DigestAlgorithm, str: String, options: Dige
   }
 }
 
+private func getRandomValues(array: TypedArray) throws -> TypedArray {
+  let status = SecRandomCopyBytes(
+    kSecRandomDefault,
+    array.byteLength,
+    array.rawPointer
+  )
+
+  if status == errSecSuccess {
+    return array
+  } else {
+    throw FailedGeneratingRandomBytesException(status)
+  }
```

Comment:
This would be more Swift-y 👇 
```suggestion
  guard status == errSecSuccess else {
    throw FailedGeneratingRandomBytesException(status)
  }
  return array
```